lib/logstash/outputs/elasticsearch.rb

```diff
@@ -8,32 +8,20 @@
 require "socket" # for Socket.gethostname
 require "thread" # for safe queueing
 require "uri" # for escaping user input
-require 'logstash-output-elasticsearch_jars.rb'
+require "logstash/outputs/elasticsearch/http_client"
 
 # This output lets you store logs in Elasticsearch and is the most recommended
 # output for Logstash. If you plan on using the Kibana web interface, you'll
-# need to use this output.
+# want to use this output.
 #
-# *VERSION NOTE*: Your Elasticsearch cluster must be running Elasticsearch 1.0.0 or later.
+# This output only speaks the HTTP, which is the preferred protocol for interacting with elasticsearch. By default
```

Comment:
For sure! (/me remembers when in all official copy Elasticsearch was always lowercased)